### PR TITLE
Feat: dark mode fix fix

### LINKED ISSUE #15: Add dark mode
## Description

Self-explanatory. Add dark mode to all pages.

## Implementation Model

Use the chakra-ui dark mode feature. Add a toggle button in the upper right of the page.

### PR DESCRIPTION
**Pull request by kelilipan**
This resolve #15
related to #18 
## Main Page
image

## Hospital list page
image

## Map Page
image
image

### CODE REVIEW

File: components/HospitalCard.js [hunk 1]
```diff
@@ -41,11 +59,15 @@ export default function HospitalCard(props) {
...
+            bgColor={
+              hospital.available_bed > 0
+                ? bgDistanceAvailable
+                : bgDistanceNotAvailable
+            }
             borderRadius="md"
           >
             <Text fontSize={['sm', 'md']} fontWeight="bold">
-              {hospital.distance.toFixed(2)} KM
+              {hospital.distance?.toFixed(2)} KM
```
Comment (kelilipan) [inline components/HospitalCard.js]: i'm adding this because if we didn't grant location access, this cause an error
image

Comment (agallio) [reply components/HospitalCard.js]: Oh! Wait checking

**Comment by agallio** [comment]
Please merge the main branch, @svspicious. Thank you for the good catch, my bad 🙇🏻

**Comment by kelilipan** [comment]
Ready to merge i think

File: components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { useRouter } from 'next/router'
+
+const Navbar = () => {
+  const { colorMode, toggleColorMode } = useColorMode()
+  const isDarkMode = colorMode === 'dark'
+  const bgButton = isDarkMode ? 'gray.700' : 'gray.100'
+  const router = useRouter()
+  const isMapPage = router.pathname === '/map'
+  return isMapPage ? null : (
+    <Box pos="fixed" top={0} right={0} p={2} zIndex="modal">
```
Comment (agallio) [inline components/Navbar.js]: Please add `borderRadius="md"` to this. 🙇🏻

**Comment by agallio** [comment]
Niceeee. Let's go 🚢

**Review by agallio — APPROVED** [review]

**Pull request merged**